schema/node.json

```diff
@@ -4,21 +4,24 @@
   "x-tags": [
     "hyperion"
   ],
-  "description": "Represents a hyperion JSON object",
+  "description": "An array of items",
```

Comment:
I would keep it closer to the original description and maybe say "Represents a resource"